webpack.config.js

```diff
@@ -106,14 +127,15 @@ const wordpressExternals = ( context, request, callback ) =>
  *
  * @see {@link https://webpack.js.org/configuration/configuration-types/#exporting-a-function}
  *
- * @param {object}  env                              additional config options
- * @param {boolean} env.externalizeWordPressPackages whether to bundle or extern the `@wordpress/` packages
- * @param {object}  argv                             given by webpack?
+ * `env` params are typically set by files in `bin/sdk/*`
+ *
+ * @param {object}  env                               environment options
+ * @param {boolean} env.externalizeWordPressPackages  whether to bundle or extern the `@wordpress/` packages
+ * @param {string}  env.stylesNamespacing             prefix Calypso component styles with CSS class or ID
```

Comment:
probably singular might work more naturally here…

```js
/**
 * @param {string} styleNamespace prefix Calypso component styles with this class or ID
 */
```

also of not here leading back to my earlier comment - the explanation is that this applies to Calypso components but we're only excluding specific directories. is it more difficult to whitelist directories instead? if the exclude-list is better then we should probably update this comment to make that less confusing